u-boot/include/soc/qca_soc_common.h

```diff
@@ -2012,7 +2012,7 @@
 	#define CONFIG_QCA_PLL_IN_FLASH_MAGIC_OFFSET	\
 				(CFG_FLASH_BASE + \
 				 CONFIG_QCA_PLL_IN_FLASH_BLOCK_OFFSET + \
-				 0x0000FFE0)
+				 0x0000FFD0)
```

Comment:
can you explain what this change does? from my memory, there is a header in PLL flash block are you trying to exclude it?

Reply:
Hello, thank you for reviewing. I'm no longer working on it, sorry for late response.

I put pll in a lower offset to fit in a PLL block (0x10000 bytes), at the end. qca956x uses 9 registers (qca955x only five). `struct clk_cfg_flash` is now 0x28 bytes, instead of 0x18 bytes

Reply:
ok